### PR TITLE
Feature/1281 mozfest landing page

### PR DESCRIPTION
**Pull request by kevinhowbrook**
# Description

This PR will add a new MozfestLandingPage.

For now it will mimic the moz fest primary page, however this will divert in upcoming tickets to offer a modified hero and different streamfield blocks. 

Designs (if you are curious) are here https://www.figma.com/file/qfVqiUyj10U1jL6Km50YIh/MozFest-2024?type=design&node-id=430-17031&mode=design&t=psARyENoB5UP4sLh-0

Link to sample test page: This can be viewed locally on http://mozfest.localhost:8000/en/landing-page-2/ - I added this into the fake data process by building a factory as per other pages.

Related PRs/issues: https://torchbox.monday.com/boards/1334760528/pulses/1337351281

# Checklist

**Tests**
- [x] Is the code I'm adding covered by tests?

**Changes in Models:**
- [x] Did I update or add new fake data?
- [ ] Did I squash my migration?
- [ ] Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

**Documentation:**
- [x] Is my code documented?
- [ ] Did I update the READMEs or wagtail documentation?

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**

### CODE REVIEW

File: env.default [hunk 1]
```diff
@@ -50,7 +50,7 @@ BASKET_URL=https://basket-dev.allizom.org
 # CSP config
 CSP_CHILD_SRC=" 'self' https://www.youtube.com https://www.youtube-nocookie.com "
 CSP_CONNECT_SRC=" * "
-CSP_DEFAULT_SRC=" 'none' "
+CSP_FRAME_ANCESTORS=" 'self' "
```
Comment (kevinhowbrook) [inline env.default:53]: Just noticed this is still not added as default, so fixed it whilst here - related to https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10768

Comment (tbrlpld) [reply env.default:53]: Should we be removing `CSP_DEFAULT_SRC=" 'none' "` to add this?

File: network-api/networkapi/mozfest/factory.py [hunk 2]
```diff
@@ -4,13 +4,12 @@
 from wagtail.models import Site as WagtailSite
 from wagtail_factories import PageFactory
 
+from networkapi.mozfest import models as mozfest_models
```
Comment (tbrlpld) [inline network-api/networkapi/mozfest/factory.py:7]: 🤌

**Review by tbrlpld — APPROVED** [review]

File: env.default (hunk 1 above)
Comment (kevinhowbrook) [reply env.default]: :facepalm: wrong line, my bad

Comment (kevinhowbrook) [reply env.default]: ea005e40e217a81e5186210f11046d2d78a760fe

**Comment by kevinhowbrook** [comment]
> Something weird is happening. I cannot add a landing page to the home page. When I click to do so, I just land on the dashboard in the admin.
> 
> Screen Recording 2023-12-08 at 10 02 56 AM [ Screen Recording 2023-12-08 at 10 02 56 AM ](https://private-user-images.githubusercontent.com/24797493/289153086-b929f753-3161-4a28-806a-852d58801041.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyOTAxMTksIm5iZiI6MTcwMjI4OTgxOSwicGF0aCI6Ii8yNDc5NzQ5My8yODkxNTMwODYtYjkyOWY3NTMtMzE2MS00YTI4LTgwNmEtODUyZDU4ODAxMDQxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDEwMTY1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxZjZiNjY2MmFiNWEwZmQ4NDdjNmIxMzg5ZmY1NjUxMjgxMzNmOTdjYTY2YTY1ZTEyMGE2OTQ3ZmIyNmJmMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2YRg7A--Etw4R3HKgEX3FEhesg6xVXTk-F9bWNjcyQI) [ ](https://private-user-images.githubusercontent.com/24797493/289153086-b929f753-3161-4a28-806a-852d58801041.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyOTAxMTksIm5iZiI6MTcwMjI4OTgxOSwicGF0aCI6Ii8yNDc5NzQ5My8yODkxNTMwODYtYjkyOWY3NTMtMzE2MS00YTI4LTgwNmEtODUyZDU4ODAxMDQxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDEwMTY1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxZjZiNjY2MmFiNWEwZmQ4NDdjNmIxMzg5ZmY1NjUxMjgxMzNmOTdjYTY2YTY1ZTEyMGE2OTQ3ZmIyNmJmMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2YRg7A--Etw4R3HKgEX3FEhesg6xVXTk-F9bWNjcyQI)

I _think_ you must have been behind a commit when reviewing because I found that too and had to add https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11513/files#diff-163e2e63ed52735b8f4851ade9367ba7ce386af7420053dedb35c3233d330131R238

**Comment by kevinhowbrook** [comment]
> Something weird is happening. I cannot add a landing page to the home page. When I click to do so, I just land on the dashboard in the admin.
> 
> Screen Recording 2023-12-08 at 10 02 56 AM [ Screen Recording 2023-12-08 at 10 02 56 AM ](https://private-user-images.githubusercontent.com/24797493/289153086-b929f753-3161-4a28-806a-852d58801041.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyOTAxMTksIm5iZiI6MTcwMjI4OTgxOSwicGF0aCI6Ii8yNDc5NzQ5My8yODkxNTMwODYtYjkyOWY3NTMtMzE2MS00YTI4LTgwNmEtODUyZDU4ODAxMDQxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDEwMTY1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxZjZiNjY2MmFiNWEwZmQ4NDdjNmIxMzg5ZmY1NjUxMjgxMzNmOTdjYTY2YTY1ZTEyMGE2OTQ3ZmIyNmJmMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2YRg7A--Etw4R3HKgEX3FEhesg6xVXTk-F9bWNjcyQI) [ ](https://private-user-images.githubusercontent.com/24797493/289153086-b929f753-3161-4a28-806a-852d58801041.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyOTAxMTksIm5iZiI6MTcwMjI4OTgxOSwicGF0aCI6Ii8yNDc5NzQ5My8yODkxNTMwODYtYjkyOWY3NTMtMzE2MS00YTI4LTgwNmEtODUyZDU4ODAxMDQxLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDEwMTY1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxZjZiNjY2MmFiNWEwZmQ4NDdjNmIxMzg5ZmY1NjUxMjgxMzNmOTdjYTY2YTY1ZTEyMGE2OTQ3ZmIyNmJmMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2YRg7A--Etw4R3HKgEX3FEhesg6xVXTk-F9bWNjcyQI)

This should be fixed now, I think you might have been behind a commit that declared the page types https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11513/files#diff-163e2e63ed52735b8f4851ade9367ba7ce386af7420053dedb35c3233d330131R238

**Comment by kevinhowbrook** [comment]
Needs rebase

**Pull request merged**